spec/main_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './spec/spec_helper'
```

Comment:
Looks like for other projects we put this in a separate file that gets loaded in for all tests.

Here's the docs - https://relishapp.com/rspec/rspec-core/v/3-5/docs/configuration/read-command-line-configuration-options-from-files

So if you add a `.rspec` file, and then add `--require spec_helper` to it, think that should do it